pkg/kubelet/kubelet.go

```diff
@@ -1723,13 +1725,47 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 			}
 		}
 
-		pods, mirrorPods := kl.GetPods()
+		pods, mirrorPods := kl.podManager.GetPodsAndMirrorMap()
 		if err := handler.SyncPods(pods, podSyncTypes, mirrorPods, start); err != nil {
 			glog.Errorf("Couldn't sync containers: %v", err)
 		}
 	}
 }
 
+// syncStatus syncs pods statuses with the apiserver. Spread the updates over the specified deadline.
+func (kl *Kubelet) syncStatus(deadline time.Duration) {
```

Comment:
How did this sneak in? I think this was removed by an earlier PR today. Bad merge by git?